index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Metaverse Survey Form</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    
+
+    <section id="header">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+                    <h2>Microverse Survey Form</h2>
+                    <p>Fill in every field correctly</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
```

Comment:
please use a semantic HTML tags in the header like <header> and <nav> instead of the section for better accessibility and SEO

Reply:
- [ ] Good job using inline comments👍 Please kindly ensure the comment is clear and easy to understand and should be in one line. This helps to keep the feedback concise and focused.😊

Reply:
I am sorry but I have tried and searched a lot but don't know how to make the comments stack above each other like that, so could you assist me? @Johnadibe